classes/blocks/class-carouselheader.php

```diff
@@ -136,12 +136,14 @@ public function prepare_data( $fields ): array {
 		// Enqueue js for the frontend.
 		if ( ! $this->is_rest_request() ) {
 			wp_enqueue_script( 'hammer', 'https://cdnjs.cloudflare.com/ajax/libs/hammer.js/2.0.8/hammer.min.js', [], '2.0.8', true );
+			wp_register_script( 'lazyload', 'https://cdn.jsdelivr.net/npm/vanilla-lazyload@11.0.4/dist/lazyload.min.js', [], '1.0.4', true );
```

Comment:
can you do the same CDN change here as in master-theme?

Reply:
Done!